testsupport/wait/awaitility.go

```diff
@@ -41,7 +41,7 @@ import (
 
 const (
 	DefaultRetryInterval             = time.Millisecond * 100 // make it short because a "retry interval" is waited before the first test
-	DefaultTimeout                   = time.Second * 120
+	DefaultTimeout                   = time.Second * 240
```

Comment:
Any particular reason for increasing the default timeout?

Reply:
that was not intended to remain in the PR. 
I got some weird issues where a test would timeout while waiting for a space to be provisioned, and when I checked the space a few minutes later, it would be actually provisioned 🤷‍♂️ 

Anyways, I will revert this change.